halyard-config/src/main/java/com/netflix/spinnaker/halyard/validate/v1/ValidateFieldRegex.java

```diff
@@ -33,7 +33,7 @@ protected ValidateFieldRegex(String subject, String pattern, String patternDescr
   public Stream<String> validate() {
     boolean matches = Pattern.matches(pattern, subject);
     if (!matches) {
-      return Stream.of(String.format("%s (\"%s\" failed to match %s)", patternDescription, subject, pattern));
+      return Stream.of(String.format("%s (\"%s\" failed to match \"%s\")", patternDescription, subject, pattern));
```

Comment:
nit: Are the double quotes inside truly necessary? It would make the string easier to read if you used single, unescaped quotes:

`String.format("%s ('%s' failed to match '%s')"`